src/ProjectTemplates/test/Infrastructure/GenerateTestProps.targets

```diff
@@ -15,6 +15,7 @@
         MicrosoftAspNetCoreAppRuntimePackageVersion=$(SharedFxVersion);
         SupportedRuntimeIdentifiers=$(SupportedRuntimeIdentifiers.Trim());
         DefaultNetCoreTargetFramework=$(DefaultNetCoreTargetFramework);
+        KnownAppHostPackOrFrameworkReferenceTfm=$(KnownAppHostPackOrFrameworkReferenceTfm);
```

Comment:
@jkotalik or @wtgodbe could one of you please duplicate this change in the other copy of GenerateTestProps.targets and src/ProjectTemplates/BlazorTemplates.Tests/Infrastructure/TemplateTests.props.in❔

In addition, please look at the template tests running on Helix. I _think_ the right thing to do is split the following bit into another file that's only used in local tests and include the remainder of the generated TemplateTests.props file in the Helix content.

https://github.com/dotnet/aspnetcore/pull/26788/files#diff-2b647fd5676115c1ec8d4bb4fb9bca0c60012fe352639a26b467b249724a151cR47-R56